src/mavsdk/plugins/mission_raw/mission_import.cpp

```diff
@@ -219,4 +320,66 @@ MissionImport::import_complex_mission_items(const Json::Value& json_item)
     return {mission_items};
 }
 
+std::vector<MissionRaw::MissionItem>
+MissionImport::import_polygon_geofences(const Json::Value& polygons)
+{
+    std::vector<MissionRaw::MissionItem> polygon_geofences;
+
+    for (const auto& polygon : polygons) {
+        bool inclusion = polygon["inclusion"].asBool();
+        const auto& points = polygon["polygon"];
+        for (const auto& point : points) {
+            MissionRaw::MissionItem item{};
+
+            item.command = inclusion ? MAV_CMD_NAV_FENCE_POLYGON_VERTEX_INCLUSION :
+                                       MAV_CMD_NAV_FENCE_POLYGON_VERTEX_EXCLUSION;
+            item.frame = MAV_FRAME_GLOBAL;
+            item.param1 = set_float(points.size());
+            item.x = set_int32(point[0]);
+            item.y = set_int32(point[1]);
+            item.mission_type = MAV_MISSION_TYPE_FENCE;
+
+            polygon_geofences.push_back(item);
+        }
+    }
+
+    return polygon_geofences;
+}
+
+std::vector<MissionRaw::MissionItem>
+MissionImport::import_circular_geofences(const Json::Value& circles)
+{
+    std::vector<MissionRaw::MissionItem> circular_geofences;
+
+    for (const auto& circle : circles) {
+        bool inclusion = circle["inclusion"].asBool();
```

Comment:
What should we do if there is no field `inclusion`? I think we should default to `inclusion = true` instead, no?

Reply:
Okay. If the field is missing, I'll make it default to true.